scripts/addVersion.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
The shebang should only be in here if we ever plan on executing this script directly, which I don't think is the case. @mikkoh can correct me if I'm wrong.

As for the `use strict`, I think that's to satisfy some versions of node around the usage of `const` and `let`. If you know a way around that, please let me know. It's super annoying.